test/Quake/state_prep.qke

```diff
@@ -0,0 +1,114 @@
+// ========================================================================== //
+// Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                 //
+// All rights reserved.                                                       //
+//                                                                            //
+// This source code and the accompanying materials are made available under   //
+// the terms of the Apache License 2.0 which accompanies this distribution.   //
+// ========================================================================== //
+
+// RUN: cudaq-opt -state-prep %s | FileCheck %s
```

Comment:
```suggestion
// RUN: cudaq-opt -state-prep -canonicalize %s | FileCheck %s
```
This would simplify the output and reduce the number of lines, I think.

Reply:
will do, thanks

Reply:
done